packages/publisher/src/publish-registry.ts

```diff
@@ -233,11 +235,20 @@ interface Registry {
 async function generateRegistry(typings: readonly TypingsData[]): Promise<Registry> {
   return {
     entries: Object.fromEntries(
-      await Promise.all(
-        typings.map(async (typing) => [
-          typing.typesDirectoryName,
-          filterTags((await pacote.packument(typing.name, { cache: cacheDir }))["dist-tags"]),
-        ]),
```

Comment:
Might be wrong but I'm pretty sure this was attempting to request all 8000 some packuments at once?

Reply:
seems like it